src/main/java/com/minecolonies/coremod/placementhandlers/MinecoloniesPlacementHandlers.java

```diff
@@ -155,9 +155,11 @@ public Object handle(
                 {
                     handleTileEntityPlacement(tileEntityData, world, pos);
                 }
+
+                entity = world.getTileEntity(pos);
```

Comment:
this isnt needed, same as above

Reply:
Unless a new tileEntity was placed, right?

Reply:
oh you're right, better leave it then

Reply:
It grows quadratic as well, so it should be fine